lib/jobs/isc-dhcp-lease-poller.js

```diff
@@ -157,6 +161,18 @@ function iscDhcpLeasePollerFactory(
                 return ip;
             }
 
+            if(line.startsWith('ends')) {
+                // slice off the to just get the date and time
+                var endTime = line.slice(7, -1);
+
+                var leaseDate = new Date(endTime);
+
+                //Check to see if it is not expired.
+                if(leaseDate > todaysDate) {
+                    notExpired = true;
```

Comment:
Actually, is this variable even necessary? Couldn't you just `return ip` here if the conditional is `leaseDate < todaysDate`?